lua/includes/modules/greatzenkakuman/predicted.lua

```diff
@@ -3,12 +3,6 @@ module("greatzenkakuman.predicted", package.seeall)
 
 local IsValid = IsValid
 local CurTime = CurTime
-local isentity = isentity
-local istable = istable
-local isvector = isvector
-local isangle = isangle
-local ismatrix = ismatrix
-local pairs = pairs
```

Comment:
Same as the main script but I write a comment here, too; I wonder why these localization has been removed because I assume that basically localizing global functions is a good practice for performance.